### ISSUE TITLE
False positive in rule `vertical_parameter_alignment_on_call `

### ISSUE THREAD

**Issue by Jeehut** [realm/SwiftLint#4273]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

In the following code, the rule `vertical_parameter_alignment_on_call ` is giving me a warning where I would not expect to see one:

```swift
.alert(
   "Purchase successful",
   isPresented: .constant(self.viewStore.successfulPurchaseDialogText != nil)
) {
   Button("Continue") {
      self.viewStore.send(.workflowCompleted)
   }
} message: {
   Text(self.viewStore.successfulPurchaseDialogText ?? "")
}
```

### Environment

Version 0.49.1
Installed vie Homebrew

**Comment by 417-72KI** [comment]
Same here.
[Multiple trailing closures](https://github.com/apple/swift-evolution/blob/main/proposals/0279-multiple-trailing-closures.md) should not be triggered.
